### PR TITLE
Webapp ify script

### PR DESCRIPTION
**Pull request by ebuckley**
Had a few moments in the Petone library, and decided to mash together a little webapp implementation of the conversion script.

To test it out, simply pull the branch down, `npm install` then `npm start`. The webapp will be running 
 at http://localhost:3000

### CODE REVIEW

**Comment by ebuckley** [comment]
At the moment, the saved endpoints are just stored in memory, any api endpoints will only exist for the time that the process is running.

File: static/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            <input required type="text" name='source_url' placeholder='<yoursite>/data.json'/>
+            </label>
+          </div>
+          <div class="field">
+            <label>
+              Source CSV
+              <input required type="file" name="csv_file" id="">
+            </label>
+          </div>
+          <button class='ui button primary' type="submit">Create a dcat API endpoint</button>
```
Comment (camfindlay) [inline static/index.html]: What we want them to do is download the resulting json file and place it on their own hosting space. If you can remove the preamble here and replace with clean instructions we might be able to look at making available as a microservice hosted somewhere. It's a useful feature that will help agencies that aren't able to run a node app locally.

**Comment by ebuckley** [comment]
I've removed a lot of the original content and simplified the instructions

When you submit the form it will be downloaded directly now.

**Comment by camfindlay** [comment]
Testing now

**Comment by camfindlay** [comment]
LGTM...

**Pull request merged**